src/azure-cli/azure/cli/command_modules/rdbms/flexible_server_custom_mysql.py

```diff
@@ -41,98 +41,88 @@ def flexible_server_create(cmd, client, resource_group_name=None, server_name=No
     storage_mb *= 1024
 
     from azure.mgmt.rdbms import mysql_flexibleservers
-    try:
-        db_context = DbContext(
-            azure_sdk=mysql_flexibleservers, cf_firewall=cf_mysql_flexible_firewall_rules, cf_db=cf_mysql_flexible_db,
-            logging_name='MySQL', command_group='mysql', server_client=client)
-
-        # Raise error when user passes values for both parameters
-        if subnet_arm_resource_id is not None and public_access is not None:
-            raise CLIError("Incorrect usage : A combination of the parameters --subnet "
-                           "and --public_access is invalid. Use either one of them.")
-
-        # When address space parameters are passed, the only valid combination is : --vnet, --subnet, --vnet-address-prefix, --subnet-address-prefix
-        # pylint: disable=too-many-boolean-expressions
-        if (vnet_address_prefix is not None) or (subnet_address_prefix is not None):
-            if (((vnet_address_prefix is not None) and (subnet_address_prefix is None)) or
-                    ((vnet_address_prefix is None) and (subnet_address_prefix is not None)) or
-                    ((vnet_address_prefix is not None) and (subnet_address_prefix is not None) and
-                     ((vnet_resource_id is None) or (subnet_arm_resource_id is None)))):
-                raise CLIError("Incorrect usage : "
-                               "--vnet, --subnet, --vnet-address-prefix, --subnet-address-prefix must be supplied together.")
-
-        server_result = firewall_id = subnet_id = None
-
-        # Check availability for server name if it is supplied by the user
-        if server_name is not None:
-            check_name_client = cf_mysql_check_resource_availability(cmd.cli_ctx, None)
-            server_availability = check_name_client.execute(server_name, DELEGATION_SERVICE_NAME)
-            if not server_availability.name_available:
-                raise CLIError(server_availability.message)
```

Comment:
Why delete the check for server name?

Reply:
It's duplicated and causing error. The name check will be done inside _create_server function.